From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Thu, 29 Aug 2024 00:51:43 +0700
Subject: [PATCH] interfaces: kill android.hardware.audio.sounddose-vendor-impl

E audiohalservice: Failed to dlopen android.hardware.audio.sounddose-vendor-impl.so: dlopen failed: library "android.hardware.audio.sounddose-vendor-impl.so" not found

but then you

FAILED:
build/make/core/main.mk:1377: warning:  device/xiaomi/blossom/lineage_blossom.mk includes non-existent modules in PRODUCT_PACKAGES
Offending entries:
android.hardware.audio.sounddose-vendor-impl

if you are optional then please kill yourself, thanks

Change-Id: I6b2439bba24eb2ca649d39fbe477c9e22c83990f
Signed-off-by: fukiame <fukiame@proton.me>

diff --git a/audio/common/all-versions/default/service/service.cpp b/audio/common/all-versions/default/service/service.cpp
index 1557a95..4989619 100644
--- a/audio/common/all-versions/default/service/service.cpp
+++ b/audio/common/all-versions/default/service/service.cpp
@@ -138,10 +138,6 @@ int main(int /* argc */, char* /* argv */ []) {
             "android.hardware.bluetooth.audio-impl",
             "createIBluetoothAudioProviderFactory",
         },
-        {
-            "android.hardware.audio.sounddose-vendor-impl",
-            "createISoundDoseFactory",
-        },
     };
     // clang-format on
 
